doc/source/setup-jupyterhub.rst

```diff
@@ -40,11 +40,22 @@ For the following steps, use your favorite code editor. We'll use the
    .. code-block:: yaml
 
       proxy:
-        secretToken: "RANDOM_STRING"
+        secretToken: "<OUTPUT-OF-`openssl rand -hex 32`>"
 
 .. Don't put an example here! People will just copy paste that & that's a security issue.
 
-4. Save the ``config.yaml`` file. If using the nano editor, hit ``Ctrl-X`` and
+4. **Azure AKS only** If you're on Microsoft Azure AKS, you must disable
+   RBAC. Do so by putting the following in ``config.yaml``
+
+   .. code-block:: yaml
+
+      rbac:
+         enabled: false
+
+   See the `RBAC documentation <security.html#use-role-based-access-control-rbac>`_
+   for more details.
+
+5. Save the ``config.yaml`` file. If using the nano editor, hit ``Ctrl-X`` and
```

Comment:
Do we have nano specific commands in other places? If not IMO we should remove that from here too.